src/core/config/config.service.ts

```diff
@@ -117,6 +117,18 @@ export class ConfigService implements EmailOptionsFactory {
     };
   }
 
+  sendProgressReportNotifications = this.env
+    .boolean('PROGRESS_REPORT_NOTIFY_EMAILS')
+    .optional(false);
```

Comment:
Default to `EMAIL_SEND` so it doesn't need to be manually specified in most cases.
```suggestion
    .optional(this.createEmailOptions().send);
```